src/test/ui/hygiene/no_implicit_prelude.rs

```diff
@@ -1,7 +1,7 @@
 #![feature(decl_macro)]
 
 mod foo {
-    pub macro m() { Vec::new(); ().clone() }
+    pub macro m() { Vec::<i32>::new(); ().clone() }
```

Comment:
We don't *need* this qualification per se, but it will avoid type inference errors if some parts of the code successfully compile in the future. I can remove it if you want though.